100.md

```diff
@@ -0,0 +1,78 @@
+
+Although there are currently mechanisms to protect and safeguard our keys, this does not provide complete reliability, and they can always be stolen. To try to mitigate the impact of this problem, this NIP provides a mechanism by which once the key owner suspects or has certainty that the keys have been stolen, they can lock their user and thus prevent (or at least partially prevent) events from being emitted in their name.
+
+### User Lock Event
+
+#### Field `kind`
+
+The `kind:1000` event is defined, which will be sent to the Relays and will indicate that this user should be locked.
+
+#### Field `pubkey`
+
+This is the public key of the event signer and the corresponding public key associated with the stolen private key that wants to be locked.
+
+#### Field `content`
+
+The `content` field may include a description of the reason why the user wants to be locked, although it can be left empty.
+
+##### Example
+
+```json
+{
+  "kind": 1000,
+  "pubkey": <pubkey to be locked>,  
+  "content": "Someone has stolen my password",
+  ...
+}
+```
+### Relay Usage
+
+Relays that implement this NIP, once they receive the `1000` type event, MUST NOT accept any other event issued by the user who has been locked.
+
+Previous events CAN still be queried, but NO operations on them, such as deletion ([NIP-09](./09.md)) or replacement (in the case of replaceable events), will be allowed.
+
+### Client Usage
+
+To lock a user, clients will have an option that allows performing this action. This action will emit a `kind:1000` event signed with the private key that is to be locked.
+
+Clients that implement this NIP MUST check if a `kind:1000` event has been issued. If so, they MUST either hide the events of that user or indicate through some visual mechanism that the user authoring those events has been locked.
+
+Optionally, clients CAN display the `1000` locking event, indicating the reason included in the `content` field. If this field is empty, they can display a generic message. They can also indicate in the user's profile that the user has been locked.
+
+#### Optional
+
+Optionally, clients CAN prevent the sending of events signed by a locked user, as well as decrypt direct encrypted messages using [NIP-04](./04.md) or [NIP-17](./17.md).
+
+For the key thief, this is easily avoidable by using another client or developing a custom one, which is why it is defined as an optional feature in this NIP. However, any difficulty in the illicit use of a key will be welcomed.
+
+
+
+### NIP-59 Usage
+
+The operation of the Relays and C
+lients in the event of receiving an event using [NIP-59](./59.md) will be the same as described in the previous sections, except that the signer of the `seal` event (`kind:13`) will be used to validate whether the event should be accepted (in the case of Relays) or hidden/marked (in the case of Clients).
```

Comment:
This is impossible for relays to enforce if they don't have the key to decrypt the wrapper.

Reply:
Yes, you're right. My mistake. I will remove this part.

Reply:
Done in 45b863d2c3b6bd8c2f7808d71f1bdf2fb987f53f

Reply:
> Yes, you're right. My mistake. I will remove this part.

Done in 45b863d2c3b6bd8c2f7808d71f1bdf2fb987f53f